package.json

```
@@ -67,7 +67,7 @@
     "ngprogress": "~1.1.3",
     "ngstorage": "~0.3.11",
     "numeral": "~2.0.6",
-    "object.values": "~1.1.1",
+    "object.values": "1.1.6",
```

Let's mark this here as the open one and the resolution below as the lockdown.

```suggestion
    "object.values": "~1.1.6",
```

---

I tried leaving that in, but for some reason that doesn't work.  This dependency is on a separate line from the indirect dependency on `object.values` in yarn.lock.  If I leave the `~` I get the following error:
```
root@aebcb0cce0fa manageiq-ui-service]# yarn run build
yarn run v1.22.18
$ yarn build:git-hash && NODE_ENV="production" webpack --bail --progress --profile --config config/webpack.prod.js
$ node config/githash.js
Successfully wrote Git hash - cab1c3e4d55d0fd69e91594500ade08c87d56bf2
Backend proxied on http://[::1]:3000
10% building 0/1 entries 0/0 dependencies 0/0 modules(node:75577) DeprecationWarning: Hook.context is deprecated and will be removed
(Use `node --trace-deprecation ...` to show where the warning was created)
[webpack-cli] ModuleNotFoundError: Module not found: Error: Can't resolve 'es-abstract/2023/RequireObjectCoercible' in '/root/BUILD/manageiq-ui-service/node_modules/object.values'
    at /root/BUILD/manageiq-ui-service/node_modules/webpack/lib/Compilation.js:2016:28
    at /root/BUILD/manageiq-ui-service/node_modules/webpack/lib/NormalModuleFactory.js:805:13
    at eval (eval at create (/root/BUILD/manageiq-ui-service/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
    at /root/BUILD/manageiq-ui-service/node_modules/webpack/lib/NormalModuleFactory.js:272:22
    at eval (eval at create (/root/BUILD/manageiq-ui-service/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at /root/BUILD/manageiq-ui-service/node_modules/webpack/lib/NormalModuleFactory.js:436:22
    at /root/BUILD/manageiq-ui-service/node_modules/webpack/lib/NormalModuleFactory.js:118:11
    at /root/BUILD/manageiq-ui-service/node_modules/webpack/lib/NormalModuleFactory.js:677:25
    at /root/BUILD/manageiq-ui-service/node_modules/webpack/lib/NormalModuleFactory.js:881:8
    at /root/BUILD/manageiq-ui-service/node_modules/webpack/lib/NormalModuleFactory.js:1001:5
resolve 'es-abstract/2023/RequireObjectCoercible' in '/root/BUILD/manageiq-ui-service/node_modules/object.values'
  Parsed request is a module
  using description file: /root/BUILD/manageiq-ui-service/node_modules/object.values/package.json (relative path: .)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      /root/BUILD/manageiq-ui-service/node_modules/object.values/node_modules doesn't exist or is not a directory
      /root/BUILD/manageiq-ui-service/node_modules/node_modules doesn't exist or is not a directory
      looking for modules in /root/BUILD/manageiq-ui-service/node_modules
        existing directory /root/BUILD/manageiq-ui-service/node_modules/es-abstract
          using description file: /root/BUILD/manageiq-ui-service/node_modules/es-abstract/package.json (relative path: .)
            using description file: /root/BUILD/manageiq-ui-service/node_modules/es-abstract/package.json (relative path: ./2023/RequireObjectCoercible)
              no extension
                /root/BUILD/manageiq-ui-service/node_modules/es-abstract/2023/RequireObjectCoercible doesn't exist
              .js
                /root/BUILD/manageiq-ui-service/node_modules/es-abstract/2023/RequireObjectCoercible.js doesn't exist
              as directory
                /root/BUILD/manageiq-ui-service/node_modules/es-abstract/2023/RequireObjectCoercible doesn't exist
      /root/BUILD/node_modules doesn't exist or is not a directory
      /root/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

---

If I leave out the `~` I get the following in yarn.lock (notice the version in the second one)
```
object.values@1.1.6, object.values@^1.0.3, object.values@^1.1.1:
  version "1.1.6"
  resolved "https://npm.manageiq.org/object.values/-/object.values-1.1.6.tgz#4abbaa71eba47d63589d402856f908243eea9b1d"
  integrity sha512-FVVTkD1vENCsAcwNs9k6jea2uHC/X0+JcjG8YA60FN5CMaJmG95wT9jek/xX9nornqGRrBkKtzuAu2wuHpKqvw==
  dependencies:
    call-bind "^1.0.2"
    define-properties "^1.1.4"
    es-abstract "^1.20.4"

object.values@~1.1.6:
  version "1.1.7"
  resolved "https://npm.manageiq.org/object.values/-/object.values-1.1.7.tgz#617ed13272e7e1071b43973aa1655d9291b8442a"
  integrity sha512-aU6xnDFYT3x17e/f0IiiwlGPTy2jzMySGfUB4fq6z7CV8l85CWHDk5ErhyhpfDHhrOMwGFhSQkhMGHaIotA6Ng==
  dependencies:
    call-bind "^1.0.2"
    define-properties "^1.2.0"
    es-abstract "^1.22.1"
```

---

WAT

---

yarn 1 sucks - it is what it is.